lib/charms/observability_libs/v0/kubernetes_compute_resources_patch.py

```diff
@@ -406,6 +537,7 @@ def apply(self, resource_reqs: ResourceRequirements) -> None:
         # Need to ignore invalid input, otherwise the StatefulSet gives "FailedCreate" and the
         # charm would be stuck in unknown/lost.
         if self.is_patched(resource_reqs):
+            logger.debug(f"resource requests {resource_reqs} are already patched.")
```

Comment:
In favor of readability and grep-ability:
```suggestion
            logger.debug(f"Resource requests are already patched: {resource_reqs}")
```